feature-detects/window/resizeobserver.js

```diff
@@ -0,0 +1,32 @@
+/*!
+{
+  "name": "ResizeObserver",
+  "property": "resizeobserver",
+  "caniuse": "resizeobserver",
+  "tags": ["ResizeObserver"],
+  "authors": ["Christian Andersson"],
+  "notes": [{
+    "name": "W3C Spec",
+    "href": "https://www.w3.org/TR/resize-observer/"
+  }, {
+    "name": "MDN Docs",
+    "href": "https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver"
+  }, {
+    "name": "Web.dev Article",
+    "href": "https://web.dev/resize-observer/"
+  }, {
+    "name": "Caniuse",
+    "href": "https://caniuse.com/#feat=resizeobserver"
```

Comment:
We tend not add the caniuse links as notes I left the link in the issue so you had in easily accessible when adding the caniuse metadata 🙃. For the rest it seems okay :) let's see what the maintainers think 👀

Reply:
Also, I don't know why AppVeyor lately tends to fail tests

Reply:
Is it possible to manually retrigger the CI? @MarkelFe

Reply:
@AnderssonChristian I think that no, but don't quote me on that one. Anyway, if you remove the caniuse link it will trigger again. Moreover, it doesn't matter in a PR basis, it's more that it gets annoying to see it failing.